kuwala/core/canvas/src/components/Modals/ExportBlockConfig/ExportBlockConfigBody.js

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import Icon from "../../Common/Icon";
+import ConfigurationForm from '../TransformationBlockConfig/ConfigurationForm'
+import Styles from "./ExportBlockConfigModalStyle"
+
+const ExportBlockConfigBody = ({
+       elements,
+       selectedElement,
+       setFieldValue,
+       setShowToolTip,
+       setSubmitData,
+       showToolTip,
+       values
+   }) => {
+    const renderConfigBodyHeader = () => {
+        return (
+            <div className={'flex flex-col'}>
```

Comment:
Please move everything into a styles object. Even if it's only one property. We might need to add more in the future when styling changes so it should be set up properly already with the classes. Applies to all files, but I'm not going to add a comment for every line. Please go through them regardless.